src/lib/config.ts

```diff
@@ -1,6 +1,15 @@
 import { categorizeByChecklist, categorizeHigherIsBetter, categorizeLowerIsBetter } from './scoring';
 import type { Config, DocumentationCompleteness, VersionStability, TestsData, ReleaseNotesData } from './types';
 
+/**
+ * Pillar enum - single source of truth for pillar names
+ */
+export enum Pillar {
+  MAINTENANCE = 'MAINTENANCE',
+  QUALITY = 'QUALITY',
+  POPULARITY = 'POPULARITY',
+}
```

Comment:
this is better -- keeping all hardcoded items in `config.ts`. but have you tried to not hardcode this and get it from the `Config` object? if you have and it doesn't work, what is going wrong?